pkg/trace/info/stats.go

```diff
@@ -288,6 +284,49 @@ func (s *SpansMalformed) String() string {
 	return mapToString(s.tagValues())
 }
 
+const (
+	maxAbsSamplingPriority = 10
+	numPriorityCounters    = maxAbsSamplingPriority*2 + 1
+)
+
+// samplingPriorityStats holds the sampling priority metrics that will be reported every 10s by the agent.
+type samplingPriorityStats struct {
+	// tracesPerSamplingPriority holds counters for each priority in position MaxAbsPriorityValue + priority.
+	tracesPerSamplingPriority [numPriorityCounters]int64
+}
+
+// CountSamplingPriority increments the counter of observed traces with the given sampling priority by 1
+func (s *samplingPriorityStats) CountSamplingPriority(priority sampler.SamplingPriority) {
```

Comment:
```suggestion
func (s *samplingPriorityStats) CountSamplingPriority(p sampler.SamplingPriority) {
```
Same thing here. I think there is enough use of the word "priority" on this line to make it clear what happens.

Reply:
Applied